### PR TITLE
Initial port of some Manager features

### PR DESCRIPTION
**Pull request by gre**
(will continue later with more src/hw/* files)

### CODE REVIEW

File: demo/src/demos/index.js [hunk 1]
```diff
@@ -13,5 +14,6 @@ export default {
   Derivations,
   Currencies,
   bridgestream,
-  qrstreambenchmark
+  qrstreambenchmark,
+  transports
```
Comment (meriadec) [inline demo/src/demos/index.js]: `comma-dangle` :cry:

File: demo/src/demos/transports/index.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import { setNetwork } from "@ledgerhq/live-common/lib/network";
+import genuineCheck from "@ledgerhq/live-common/lib/hw/genuineCheck";
+import getDeviceInfo from "@ledgerhq/live-common/lib/hw/getDeviceInfo";
+
+setNetwork(axios);
+
+let queue = Promise.resolve();
+const execInQueue = <T>(job: () => Promise<T>) => {
+  const p = queue.then(job);
+  queue = p;
```
Comment (meriadec) [inline demo/src/demos/transports/index.js]: don't need to use `p` ref, but I guess it's for clarity

Comment (meriadec) [reply demo/src/demos/transports/index.js]: (btw this is demo we dont care)

Comment (gre) [reply demo/src/demos/transports/index.js]: it actually is required for flow last time i checked.

Comment (gre) [reply demo/src/demos/transports/index.js]: p is of time `Promise<T>` so flow needs to remember that.

queue is a `Promise<any>` which accepts `Promise<T>`

**Pull request merged**